packages/material-ui/src/Collapse/Collapse.js

```diff
@@ -81,6 +130,16 @@ const Collapse = React.forwardRef(function Collapse(props, ref) {
     TransitionComponent = Transition,
     ...other
   } = props;
+
+  const styleProps = {
+    ...props,
+    orientation,
+    collapsedSize: collapsedSizeProp,
+    in: inProp,
```

Comment:
```suggestion
```

Seems like this will be already included in the props and it does not have a default value, so we can safely drop it. The thinking here is that, the `stylesProps` should include all props + their defaults, that is why we are spreading the props and then overriting with the default values